### PR TITLE
complete short name howto

### PR DESCRIPTION
**Pull request by tudorbarascu**

### CODE REVIEW

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 1]
```diff
@@ -643,7 +643,10 @@ for humans than a name for machine-to-machine communication.
 QGIS Server supports:
 
 * short name line edits to layers properties
+  You can change this by right clicking on a layer, choose :guilabel:`Properties` -->
+  :guilabel:`Metadata` tab --> :guilabel:`Short name` in the :guilabel:`Description` section.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: At this moment, we use :menuselection:`Properties --> Metadata` for menu and guilalbel for label in the interface (so the last two are correct).

**Comment by tudorbarascu** [comment]
rebased based on the observations

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 2]
```diff
@@ -643,7 +643,10 @@ for humans than a name for machine-to-machine communication.
 QGIS Server supports:
 
 * short name line edits to layers properties
+  You can change this by right clicking on a layer, choose :guilabel:`Properties` -->
+  :menuselection:`Metadata` tab --> :guilabel:`Short name` in the :guilabel:`Description` section.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: I was not clear, sorry: if you want to use a sequence of click, you can use one tag then embed --> inside the backquote:

:guilabel:`Properties --> Metadata tab --> Short name`

this way the `-->` will be transformed in a nice arrow.

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 3]
```diff
@@ -654,16 +657,20 @@ QGIS Server supports:
...
+
+
+* short name line edits to project properties - add a regexp validator
   ``"^[A-Za-z][A-Za-z0-9\._-]*"`` to short name line edit accessible through a
   static method
 * add a regexp validator ``"^[A-Za-z][A-Za-z0-9\._-]*"`` to short name line
   edit accessible through a static method
+
+  You can choose a short name for the project root by going to :guilabel:`Project properties` -->
+  :guilabel:`OWS Server` --> :guilabel:`Short name` in the :guilabel:`Service capabilities` section.
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: Same here that above.

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst (hunk 2 above)
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: PS: yes I changed back to guilabel, probably better than menuselection finally. Sorry.

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst (hunk 3 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: The sequence between `service capabilities` and `short name` wasn't clear to me so what about : *project root in :guilabel:\`Project properties -->  OWS Server --> Service capabilities\`  section*? Not sure if it's worth mentioning `Short name` option again but why not. 

Indeed, putting all the labels in a single guilabel (or menuselection) tag avoids separate blue backgrounded texts at the same place (and for better rendering, `-->` should be between the back quotes too)

**Comment by tudorbarascu** [comment]
how's it looking now?

**Pull request merged**